_posts/2015-09-04-channel.html

```diff
@@ -3614,6 +3614,345 @@
+				<td>401 Authorization required</td>
+				<td>The video is protected, only the owner with a valid token can access it.</td>
+			</tr>
+			<tr>
+				<td>lack_of_ownership</td>
+				<td>403 Forbidden</td>
+				<td>The given video does not belong to the user.</td>
+			</tr>
+			<tr>
+				<td>not_found</td>
+				<td>404 Not Found</td>
+				<td>The given video or caption does not exist.</td>
+			</tr>
+		</table>
+
+		<h4>Download vtt file</h4>
+		<table class="table fill-bg-light1 table-chromeless">
+			<tr>
+				<td class="verb">GET</td>
+				<td>
+					<a>https://api.ustream.tv/videos/VIDEO_ID/captions/LANGUAGE/vtt.FORMAT</a>
+					(format can be json or xml)</td>
+			</tr>
+		</table>
+
+		<h6>Response</h6>
+		<p>WEBVTT file content. Example:</p>
+		<pre><code class="json">WEBVTT
+
+00:09.000 --> 00:11.000
+We are in New York City
+</code></pre>
+
+		<h6>Error codes</h6>
+		<table class="table responsive">
+			<tr>
+				<th>error value</th>
+				<th>HTTP response code</th>
+				<th width="100%">Error conditions</th>
+			</tr>
+			<tr>
+				<td>unauthorized</td>
+				<td>401 Authorization required</td>
+				<td>The video is protected, only the owner with a valid token can access it.</td>
+			</tr>
+			<tr>
+				<td>lack_of_ownership</td>
+				<td>403 Forbidden</td>
+				<td>The given video does not belong to the user.</td>
+			</tr>
+			<tr>
+				<td>not_found</td>
+				<td>404 Not Found</td>
+				<td>The given video or caption does not exist.</td>
+			</tr>
+		</table>
+
+		<h4>Upload caption</h4>
+		<p>Create a new caption for the given video</p>
+		<table class="table fill-bg-light1 table-chromeless">
+			<tr>
+				<td class="verb">PUT</td>
+				<td>
+					<a>https://api.ustream.tv/videos/VIDEO_ID/captions/LANGUAGE/vtt.FORMAT</a>
+					(format can be json or xml)</td>
+			</tr>
+		</table>
+
+		<p>Example:</p>
+		<pre><code class="nohighlight">PUT videos/VIDEO_ID/captions/en-00.json HTTP/1.1
+Host: api.ustream.tv
+Authorization: Bearer xxxxxxxxxxxxxx
+Content-Type: text/vtt
+
+WEBVTT
+
+00:09.000 --> 00:11.000
+We are in New York City
+</code></pre>
+		<p class="note">The token in the above call is only an example.</p>
+
+		<h6>Response</h6>
+
+		<h6>Error codes</h6>
+		<table class="table responsive">
+			<tr>
+				<th>error value</th>
+				<th>HTTP response code</th>
+				<th width="100%">Error conditions</th>
+			</tr>
+			<tr>
+				<td>unauthorized</td>
+				<td>401 Authorization required</td>
+				<td>The video is protected, only the owner with a valid token can access it.</td>
+			</tr>
+			<tr>
+				<td>lack_of_ownership</td>
+				<td>403 Forbidden</td>
+				<td>The given video does not belong to the user.</td>
+			</tr>
+			<tr>
+				<td>not_found</td>
+				<td>404 Not Found</td>
+				<td>The given video or caption does not exist.</td>
+			</tr>
+			<tr>
+				<td>not_valid_vtt</td>
+				<td>400 Bad Request</td>
+				<td>Caption has some error.</td>
+			</tr>
+		</table>
+
+		<h4>Listing possible languages</h4>
```

Comment:
"List ..." legyen inkább, mert eddig nem a főnévi igenév alakot használtad, de amúgy a doksiban is össze-vissza van.. :S